war/src/main/webapp/help/system-config/systemMessage.html

```diff
@@ -1,5 +1,6 @@
 <div>
-  This message will be displayed on <a href=".">the top page</a>.
-  Useful for posting a system-wide notification to users.
-  Can contain HTML tags or whatever markup language is defined for the system.
+  This message will be displayed at the top of the
```

Comment:
no its put under the banner which is at the top of the page.
(and under any administrative alerts I think)
"This message will be displayed on the "